src/Makefile

```diff
@@ -12,14 +12,26 @@
 V=debug
 include gpu-common.mk
 
+# OpenCL include/library
+CL_HEADER_DIR:=/cygdrive/c/Program\ Files/NVIDIA\ GPU\ Computing\ Toolkit/CUDA/v10.0/include/
```

Comment:
This should come from an environment variable to where the user has this installed.

Reply:
Sure, will remove